charts/modules/infra/aws-eks-config/values.yaml

```diff
@@ -48,29 +48,32 @@ global:
 StorageClass:
   enabled: false
   items:
-    gp3:
-      name: gp3
+    gp3-persistent-125:
+      name: gp3-persistent-125
       annotations:
-        storageclass.kubernetes.io/is-default-class: "true"
+        storageclass.kubernetes.io/is-default-class: "false"
       allowVolumeExpansion: true
       provisioner: kubernetes.io/aws-ebs
       reclaimPolicy: Retain
```

Comment:
CICD Grafana uses this gp3 storageclass for persistent volume. What is the impact of renaming this? The EBS volume will remain unaffected because of the Retain policy. 

[How would the monitoring deployment behave?](https://argo-cd.luminarinfra.com/applications/infra-argo-cd/cicd-us-west-2-prod-eks-0-monitoring?resource=kind%3APersistentVolumeClaim)